fixes #11808 (tick checkbox to generate XML file) (#11830)
aspnet/AspNetCore.Docs: aspnetcore/tutorials/getting-started-with-swashbuckle.md

}
```

> Configure Swagger to use the XML file that's generated when you select the **XML documentation file** check box in the project's build properties. For Linux or non-Windows operating systems, file names and paths can be case-sensitive. For example, a *TodoApi.XML* file is valid on Windows but not CentOS.

Reviewer: With this new wording, the text becomes specific to Visual Studio. We'd like to keep the text generic enough, such that it applies elsewhere too (e.g., VS Code & VS for Mac).

Would something like the following accomplish your goal?

```suggestion
Configure Swagger to use the XML file that's generated with the preceding instructions. For Linux or non-Windows operating systems, file names and paths can be case-sensitive. For example, a *TodoApi.XML* file is valid on Windows but not CentOS.
```
